.github/workflows/tests.yml

```
@@ -26,6 +36,22 @@ jobs:
     - uses: actions/checkout@v2
     - name: setup
       run: |
-         docker pull dealii/dealii:master-focal
-         docker run -t dealii/dealii:master-focal /bin/sh -c "git clone https://github.com/kronbichler/adaflo.git && mkdir adaflo-build && cd adaflo-build && cmake ../adaflo && make -j2 adaflo && git clone https://github.com/${{ github.repository }} && cd MeltPoolDG && git fetch origin ${{ github.ref }} && git checkout FETCH_HEAD && git --no-pager  log | head -n50 && cd .. && mkdir MeltPoolDG-build && cd MeltPoolDG-build && cmake ../MeltPoolDG -D ADAFLO_LIB=/home/dealii/adaflo-build/ -D ADAFLO_INCLUDE=/home/dealii/adaflo/include/ && make -j2 && ctest -V";
+         "git clone https://github.com/MeltPoolDG/adaflo.git    && \
```

```suggestion
         command="git clone https://github.com/MeltPoolDG/adaflo.git    && \
```
This should fix the the failing adaflo CI tests. Sorry about this! Could you take a look at the failing tests without adaflo support and indent the code. I will have the current version of the code in the morning and will merge: any further improvements can be added in follow-up PRs!